src/biotite/sequence/profile.py

```diff
@@ -280,3 +291,88 @@ def _general_to_consensus(self):
         consensus = GeneralSequence(self.alphabet)
         consensus.code = np.argmax(self.symbols, axis=1)
         return consensus
+
+    def probability_matrix(self, pseudocount=0):
+        """
+        Calculate the position probability matrix (ppm) based on 'symbols'
+        and the given pseudocount.
+        This new matrix has the same shape as 'symbols'.
+
+        Parameters
+        ----------
+        pseudocount: int
+            Amount added to the number of observed cases in order to change
+            the expected probability.
+            (Default: 0)
+
+        Returns
+        -------
+        ppm: ndarray, dtype=float, shape=(n,k)
+            Calculated the position probability matrix.
```

Comment:
Did you mean to write `The calculated position probability matrix.`, or am I wrong here?